tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/ManualInstrumentation/Ci/Proxies/ITestParameters.cs

```diff
@@ -16,7 +16,7 @@ namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.ManualInstrumentation.Ci
 [DuckType("Datadog.Trace.Ci.TestParameters", "Datadog.Trace.Manual")]
 internal interface ITestParameters : IDuckType
 {
-    public Dictionary<string, object>? Metadata { get; }
+    public Dictionary<string, object?>? Metadata { get; }
 
     public Dictionary<string, object>? Arguments { get; }
```

Comment:
I wonder if this one should be `object?` too, seeing as it comes from the public API 🤔 Probably not, but may still need defensive checks

Reply:
done in [3d98a45](https://github.com/DataDog/dd-trace-dotnet/pull/6374/commits/3d98a45a5bebdc0c7de9990b062d3e8eb506713d)